buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -1815,6 +1815,58 @@ class BeamModulePlugin implements Plugin<Project> {
           }
         }
       }
+
+      project.ext.addPortableWordCountTask = { String nameSuffix, boolean isStreaming ->
```

Comment:
I'm okay to have this task in `BeamModulePlugin` for now. But my concern is that this may not be scalable if we want to add more variants (e.g. pipelines, configurations) in the future. We probably can make it more configurable or put it in separate groovy if more variants we are end up with.

Reply:
Yes, it is my concern as well, this is being discussed on the mailing list, let's continue the discussion on https://lists.apache.org/thread.html/7c48200678c570c6e42d4933f8ea4f9324d82f7ae720b591a93ffcda@%3Cdev.beam.apache.org%3E.